scrapy/utils/response.py

```diff
@@ -50,6 +51,7 @@ def response_status_message(status: Union[bytes, float, int, str]) -> str:
     return f'{status_int} {to_unicode(message)}'
 
 
+@deprecated
 def response_httprepr(response: Response) -> bytes:
```

Comment:
@wRAR FYI this function is deprecated here (and removed in 2.11) but never mentioned in the [release logs](https://docs.scrapy.org/en/2.10/news.html) (nor its removal)

Reply:
Great catch, I’ve logged https://github.com/scrapy/scrapy/issues/6207.